app/views/enrollments/index.html.erb

```diff
@@ -12,8 +12,8 @@
     </tr>
     <% @enrollments.each do |enrollment| %>
       <tr>
-        <td><%= enrollment.name%></td>
-        <td><%= enrollment.Subject %></td>
+        <td><%= enrollment.student_id%></td>
```

Comment:
enrollment.student_id won't be good to show in the UI, I would use enrollment.student.user.name